HomeTask_4.py

```diff
@@ -0,0 +1,19 @@
+"""HomeTask_4.
+
+we have four values w,x,y,z
+write if-elif-else statement that will search minimum value and print smth aka "'y' is minimum value'
+advice use python debugger and different values to check your algorithm.
+"""
+
+w, x, y, z = 100, 200, 40, 300
+
+if w < x and w < y and w < z:
+    print("'w' is the minimum value")
+elif x < w and x < y and x < z:
+    print("'x' is the minimum value")
+elif y < w and y < x and y < z:
+    print("'y' is the minimum value")
+else:
```

Comment:
Solution correct but ... looks as copy paste. If this is true or not better answer to yourself.